### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by uyen-luu**

### CODE REVIEW

File: playground/API/Identity.db
Comment (uveta) [inline playground/API/Identity.db:1]: Do we need database added to git?

Comment (uyen-luu) [reply playground/API/Identity.db:1]: @uveta Yes, we need it to use SQLite as we don't have any SQL Server.

Comment (uveta) [reply playground/API/Identity.db:1]: @uyen-luu I understand it is needed to run the application, but do we need it versioned in git? Is database file automatically created by application if it doesn't exist? Could it contain secrets or user data - usernames, passwords, salts, etc.?

Comment (uyen-luu) [reply playground/API/Identity.db:1]: The git version of IdentityDb contains some of default users which we added for testing, it these demo users would be added if no users in the IdentityDb found. If someone try our playground and register their credential for a new account, then that account would be stored for that app container only, nothing be updated to the git.

Comment (uveta) [reply playground/API/Identity.db:1]: OK, understood. Thanks for the explanation 😄

**Review by uveta — APPROVED** [review]

**Pull request merged**